jupyterlab_bigquery/src/utils/formatters.ts

```diff
@@ -0,0 +1,6 @@
+import { DateTime } from 'luxon';
+
+export function formatDate(dateString) {
+  const date = DateTime.fromISO(dateString);
+  return date.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+}
```

Comment:
Not sure if `utils/formatters` is the best place to put this - let me know if there's a more appropriate location. :)

Reply:
Sounds good to me. We can always refactor later if need be.